advntr/vntr_finder.py

```diff
@@ -660,14 +669,15 @@ def find_repeat_count_from_alignment_file(self, alignment_file, unmapped_filtere
         if len(max_flanking_repeat) < 5:
             max_flanking_repeat = []
 
-        exact_genotype = self.find_genotype_based_on_observed_repeats(covered_repeats + max_flanking_repeat)
+            exact_genotype, max_prob = self.find_genotype_based_on_observed_repeats(covered_repeats + max_flanking_repeat)
```

Comment:
I believe you put an extra indentation here so we don't get the exact_genotype for most cases.